packages/p2p-v2/src/pages/orders/screens/Orders/OrdersTable/OrdersTable.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { memo } from 'react';
+import clsx from 'clsx';
+import { TOrders } from 'types';
+import { Loader, Table, useDevice } from '@deriv-com/ui';
+import { OrdersEmpty } from '../OrdersEmpty';
+import { OrdersTableRow } from '../OrdersTableRow';
+import './OrdersTable.scss';
+
+type TOrdersTableRowRendererProps = TOrders[0];
+
+const OrdersTableRowRenderer = memo((values: TOrdersTableRowRendererProps) => <OrdersTableRow {...values} />);
+OrdersTableRowRenderer.displayName = 'OrdersTableRowRenderer';
+
+const headerRenderer = (header: string) => <span>{header}</span>;
```

Comment:
i think we should create a reusable component for this cause this is being used everywhere for each table 😆